src/main/java/org/springframework/samples/petclinic/PetClinicApplication.java

```diff
@@ -1,9 +1,11 @@
 package org.springframework.samples.petclinic;
 
+import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.builder.SpringApplicationBuilder;
 import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;
+import org.springframework.context.annotation.Bean;
```

Comment:
looks like these imports are not needed, because there are no other changes in the class 😉